tests/lib/test-common.bash.in

```diff
@@ -95,7 +95,7 @@ function check_active_process() {
 # The function is intended to be used in the setup() of
 # the docker integration tests for cc-oci-runtime.
 function kill_processes_before_start() {
-	[ "$($DOCKER_EXE ps -q)" ] && $DOCKER_EXE kill $("$DOCKER_EXE ps -q")
+	[ $($DOCKER_EXE ps -q) ] && $DOCKER_EXE kill $($DOCKER_EXE ps -q)
```

Comment:
It might be better to run the `ps -q` only once and use an explicit `[ -n $containers ]` type test. Both those are not blocking comments.

lgtm.